src/keyboard.rs

```diff
@@ -791,24 +782,20 @@ pub enum NamedKey {
     /// The Symbol modifier key (used on some virtual keyboards).
     Symbol,
     SymbolLock,
-    // Legacy modifier key. Also called "Super" in certain places.
-    Meta,
+    // Legacy modifier key.
+    Super,
     // Legacy modifier key.
     Hyper,
-    /// Used to enable "super" modifier function for interpreting concurrent or subsequent keyboard
+    /// Used to enable "meta" modifier function for interpreting concurrent or subsequent keyboard
     /// input. This key value is used for the "Windows Logo" key and the Apple `Command` or `⌘`
     /// key.
-    ///
-    /// Note: In some contexts (e.g. the Web) this is referred to as the "Meta" key.
-    Super,
+    Meta,
```

Comment:
I suggest adding that this is referred to as `Super` in some contexts. Having this in the documentation makes it easier to search.

Reply:
We could add a doc alias of "super" so that people searching the docs for super will find it?

https://doc.rust-lang.org/rustdoc/advanced-features.html#add-aliases-for-an-item-in-documentation-search